src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import './components/style.css';
+import Calculate from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculate />
```

Comment:
- [ ] Kindly note that it is required that the App component is class-based just as the Calculator component.